PayzenMulti.php

```diff
@@ -61,10 +65,14 @@ public function isValidPayment()
             $valid = $this->checkMinMaxAmount();
         }
 
+        if ($valid) {
+            $this->getDispatcher()->dispatch(new ValidationPaymentEvent(), TheliaEvents::MODULE_PAYMENT_IS_VALID);
```

Comment:
Not sure it's the good event name, I think it must be `ValidationPaymentEvent::PAYZEN_MULTI_VALIDATION_PAYEMENT`
And you do nothing with the Event object after dispatch.